torch/_inductor/config.py

```diff
@@ -148,7 +148,10 @@ def decide_compile_threads():
 
 # gemm autotuning global cache dir
 if is_fbcode():
-    global_cache_dir = "fb/cache"
+    from libfb.py import parutil
+
+    relative_path = os.path.join(__package__.replace(".", os.sep), "fb/cache")
+    global_cache_dir = parutil.get_dir_path(relative_path)
```

Comment:
```suggestion
    global_cache_dir = parutil.get_dir_path(os.path.join(__package__.replace(".", os.sep), "fb/cache"))
```

So we don't create an extra config nameed `relative_path`